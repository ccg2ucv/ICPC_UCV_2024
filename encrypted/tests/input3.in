16
A@A@KA@A@KA@A@KQRQoQRQoQRQoAIAIAAIAAIAIAXU\\_TQTQT_