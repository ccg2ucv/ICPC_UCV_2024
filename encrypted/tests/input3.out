1010;1010;1010;ABA_ABA_ABA_1919119119191HELLODADADO